win32.mak

```diff
@@ -18,7 +19,7 @@ chmgen.exe : chmgen.d
 	$(DMD) -g chmgen
 
 chm\d.hhp chm\d.hhc chm\d.hhk : chmgen.exe chm-nav-release.json $(TARGETS)
-	chmgen --target release
+	chmgen --root=$W --target release
```

Comment:
IIRC win32.mak used to build in-place, i.e. put the .html files alongside the .dd files. Has this changed?

Reply:
`win32.mak` doesn't generate the documentation anymore. This has long been deprecated, though it's `docs` for the installer: https://github.com/dlang/installer/blob/master/create_dmd_release/create_dmd_release.d#L514